src/components/ItemsListing.vue

```diff
@@ -692,6 +693,20 @@ const onClick = function (mediaItem: MediaItemType) {
   }
 };
 
+const onClear = function () {
+  params.value.search = '';
+  showSearch.value = false;
+  loadData(true);
+};
+
+const isSearchActive = function () {
```

Comment:
don't you want to make this a computed property instead of a function ?

Reply:
You're absolutely right. I'll make a change later or in the morning